config/import-control.xml

```diff
@@ -165,6 +167,8 @@
     <allow class="com.puppycrawl.tools.checkstyle.TreeWalkerAuditEvent" local-only="true"/>
     <allow class="com.puppycrawl.tools.checkstyle.TreeWalkerFilter" local-only="true"/>
     <allow class="com.puppycrawl.tools.checkstyle.XmlLoader" local-only="true"/>
+    <allow class="com.puppycrawl.tools.checkstyle.BeanPropertyType" local-only="true"/>
+    <allow class="com.puppycrawl.tools.checkstyle.PropertyType" local-only="true"/>
```

Comment:
This need to avoid errors like this:
```
[ERROR] [checkstyle] [ERROR] src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java:31:1: Disallowed import - com.puppycrawl.tools.checkstyle.BeanPropertyType. [ImportControlMain]
[ERROR] [checkstyle] [ERROR] src/main/java/com/puppycrawl/tools/checkstyle/filters/SuppressWithNearbyCommentFilter.java:32:1: Disallowed import - com.puppycrawl.tools.checkstyle.PropertyType. [ImportControlMain]
```
The package `com.puppycrawl.tools.checkstyle` is disabled by default. We need to explicitly enable it for `com.puppycrawl.tools.checkstyle.filters` and `com.puppycrawl.tools.checkstyle.checks`.

Reply:
I see my confusion now. First change is in `checks` subpackage and not in the global, top package. Can ignore.